packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -211,7 +211,7 @@ function* doMarkChatAsRead(action: ReturnType<typeof markChatAsRead>) {
     const audiusSdk = yield* getContext('audiusSdk')
     const sdk = yield* call(audiusSdk)
     const chat = yield* select((state) => getChat(state, chatId))
-    if (!chat || dayjs(chat?.last_read_at).isBefore(chat?.last_message_at)) {
+    if (!chat || !dayjs(chat?.last_read_at).isAfter(chat?.last_message_at)) {
```

Comment:
??? plz explain

Reply:
wanted to allow it to mark as read if the timestamps were equal, in case it was reading the optimistically set value

Reply:
maybe add comment for that

Reply:
I think it reads pretty straightforward in context: if the last read isn't after the last message, call chats.read